packages/patternfly-4/react-core/src/components/AboutModal/AboutModal.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import AboutModalContainer from './AboutModalContainer';
+import { canUseDOM } from 'exenv';
+import { KEY_CODES } from '../../internal/constants';
+
+const propTypes = {
+  /** content rendered inside the About Modal. */
+  children: PropTypes.node.isRequired,
+  /** additional classes added to the About Modal */
+  className: PropTypes.string,
+  /** Flag to show the  About modal */
+  isOpen: PropTypes.bool,
+  /** A callback for when the close button is clicked */
+  onClose: PropTypes.func,
+  /** Product Name */
+  productName: PropTypes.string.isRequired,
+  /** Trademark information */
+  trademark: PropTypes.string.isRequired,
+  /** the URL of the image for the Brand. */
+  brandImageSrc: PropTypes.string.isRequired,
+  /** the alternate text of the Brand image. */
+  brandImageAlt: PropTypes.string.isRequired,
+  /** the URL of the image for the Logo. */
+  logoImageSrc: PropTypes.string.isRequired,
+  /** the alternate text of the Logo image. */
+  logoImageAlt: PropTypes.string.isRequired,
+  /** the URL of the image for the Hero. */
+  heroImageSrc: PropTypes.string.isRequired,
+  /** the alternate text of the Hero image. */
+  heroImageAlt: PropTypes.string
+};
+
+const defaultProps = {
+  className: '',
+  isOpen: false,
+  onClose: () => undefined,
+  heroImageAlt: ''
+};
+
+let currentId = 0;
+
+class AboutModal extends React.Component {
+  static propTypes = propTypes;
+  static defaultProps = defaultProps;
+
+  ariaLablledbyId = `pf-about-modal-title-${currentId++}`;
+  ariaDescribedById = `pf-about-modal-content-${currentId++}`;
+
+  handleEscKeyClick = event => {
+    if (event.keyCode === KEY_CODES.ESCAPE_KEY && this.props.isOpen) {
```

Comment:
Would the listener be active if the modal was not open?

Reply:
Apparently,  I had it the other way initially (in the Modal component)  and it was updated it to have the check.

Reply:
It would. Even without the modal open this component will be mounted. The only way to not have this be the case would be to remove the listener on update if isOpen changes to false. That would probably be a better way of handling this. Also, it might be worth trying to share this entry point with the regular modal. They are both modals with different content/styles.